ravenframework/Models/PostProcessors/BasicStatistics.py

```diff
@@ -1090,7 +1090,10 @@ def _runLocal(self, inputData):
       self.raiseADebug('Starting calculate standard error on "'+metric+'"...')
       norm = stats.norm
       factor = np.sqrt(np.asarray(percent)*(1.0 - np.asarray(percent)))/norm.pdf(norm.ppf(percent))
-      sigmaAdjusted = calculations['sigma'][list(needed[metric]['targets'])]/np.sqrt(calculations['equivalentSamples'][list(needed[metric]['targets'])])
+      try:
+        sigmaAdjusted = calculations['sigma'][list(needed[metric]['targets'])]/np.sqrt(calculations['equivalentSamples'][list(needed[metric]['targets'])])
+      except:
+        sigmaAdjusted = calculations['sigma'][list(needed[metric]['targets'])]/np.sqrt(self.sampleSize)
```

Comment:
Can you put the error type in the "except"? it seems to me these changes may not be the right fix for the issue. I guess equivalentSamples may not computed for certain cases.

Reply:
Added the error type. What the right fix you have in mind?

Reply:
I guess maybe we need to figure why equivalentSamples is not calculated in certain case.

Reply:
The equivalentSamples is not computed when ```self.pbPresent``` is False.

Reply:
Can you guys do it in another MR? It is not really related to this MR. I can revert the fix if you want.

Reply:
definitely, you do not need to revert the changes. By the way, I have already approved your PR.